network/src/peer_store/browser.rs

```diff
@@ -67,15 +69,14 @@ pub struct Storage {
 impl Storage {
     pub async fn new<P: AsRef<Path>>(path: P) -> Self {
         let factory = Factory::new().unwrap();
-        let mut open_request = factory.open("network", Some(1)).unwrap();
-        let store_name = path.as_ref().to_str().unwrap().to_owned();
-        let store_name_clone = store_name.clone();
+        let database_name = path.as_ref().to_str().unwrap().to_owned();
+        let mut open_request = factory.open(&database_name, Some(1)).unwrap();
```

Comment:
There is a `secret_key` file in `network` dir. If the secret key in CKB's network directory changes, it may cause the `NodeId` to change as well. Is this expected?

Reply:
No, wasm target‘s secret_key isn't stored on this path. The user manually enters it